languages/nl/Zend_Captcha.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * EN-Revision: 30.Jul.2011
+ */
+return [
+    // Zend\Captcha\ReCaptcha
+    "Missing captcha fields" => "Missende captcha velden",
+    "Failed to validate captcha" => "Captcha validatie mislukt",
```

Comment:
Both could work, but since I don't use Zend (@mmoll asked me to translate) I don't know the context where it's used. I also considered ```Ongeldige captcha validatie``` or some version of that, but I tried to stay close to the English translation.